src/pygeoprocessing/geoprocessing.py

```diff
@@ -41,16 +41,13 @@ class ReclassificationMissingValuesError(Exception):
 
     Attributes:
         msg (str) - error message
-        missing_values (list) - a list of the missing values from the raster
-            that are not keys in the dictionary
 
     """
 
-    def __init__(self, msg, missing_values):
+    def __init__(self, msg):
         """See Attributes for args docstring."""
         self.msg = msg
-        self.missing_values = missing_values
-        super().__init__(msg, missing_values)
```

Comment:
Oh okay. Maybe stormwater should be using `utils.reclassify_raster`? I changed it because when you call `pygeoprocessing.reclassify_raster` directly, the error message has that array of missing values tacked on the end with no explanation: `pygeoprocessing.geoprocessing.ReclassificationMissingValuesError: ('The following 1 raster values [-3.4e+38] from "/Users/emily/Documents/stormwater_workspace/intermediate/lulc_aligned.tif" do not have corresponding entries in the ``value_map``: {1: 1, 2: 1, 3: 0, 4: 1, 5: 1, 6: 0, 7: 1, 8: 0, 9: 0, 10: 1, 11: 0, 12: 0, 13: 0, 14: 0}.', array([-3.4e+38], dtype=float32))`

Reply:
What do you think about this ac4b8d6? Moving the message up to the exception class, it can stay compatible with `utils.reclassify_raster` while also having a clear message by itself

Reply:
> Maybe stormwater should be using utils.reclassify_raster?

I believe the main/only reason we have that `utils` function is to better format the error message for invest users that don't know what the `value_map` refers to. So yeah, I think the stormwater model should probably use the `utils` function for consistency across invest.

Reply:
Yea, the `utils` wrapper for this function was designed with InVEST users in mind, given how often we were seeing questions on the forums related to this error message. So the `utils` wrapper is meant for developers to have more control over what would be useful for the user to know if they have an error when reclassifying.